src/AggregateFunctions/QuantileTDigest.h

```diff
@@ -335,6 +335,18 @@ class QuantileTDigest
         compress(); // Allows reading/writing TDigests with different epsilon/max_centroids params
     }
 
+    Float64 getCountEqual(Float64 value) const
+    {
+        Float64 result = 0;
+        for (const auto & c : centroids)
+        {
+            /// std::cerr << "c "<< c.mean << " "<< c.count << std::endl;
+            if (value == c.mean)
```

Comment:
So we are comparing two floats for equality? That's unlikely to work.

Reply:
yes there are other places comparing two floats https://github.com/ClickHouse/ClickHouse/blob/master/src/AggregateFunctions/QuantileTDigest.h#L143

But you are right it doesn't work of int64 or bigger integer, actually we need to make type `Value` as `template T` to make it more accurate